uiTests/src/test/resources/ide.rhino.js

```diff
@@ -41,16 +41,6 @@ function Ide() {
       });
     });
   };
-
-  this.close = function () {
-    // We can't just run `app.exit()` synchronously since then the Robot client would never receive a reply from the Robot server,
```

Comment:
This was a dumb idea from the very beginning, `Do Not Try This At Home` :hushed:

Reply:
In fact now I observe that simply sending TERM to the wrapping Gradle/xvfb-run process makes the IDE terminate more clearly than with this sh*t :hankey: